hat/assets/js/apps/Iaso/domains/forms/components/LinkToForm.tsx

```diff
@@ -1,19 +1,33 @@
 import React, { FunctionComponent } from 'react';
 
 import { Link } from 'react-router';
+import { makeStyles } from '@material-ui/core';
 import { userHasPermission } from '../../users/utils';
 import { baseUrls } from '../../../constants/urls';
 import { useCurrentUser } from '../../../utils/usersUtils';
 
 type Props = {
-    formId: string;
+    formId: string | number;
     formName: unknown;
 };
+
+const useStyles = makeStyles(theme => ({
+    link: {
+        color: theme.palette.primary.main,
```

Comment:
This will not work well in polio as the primary is pretty close to black 
![Screenshot 2022-12-15 at 11 29 17](https://user-images.githubusercontent.com/38907762/207836211-00a4b88b-9b6f-46c6-aacd-cf6a5385428f.png)